src/referencePackages/Directory.Build.targets

```diff
@@ -120,6 +120,11 @@
 
   <Target Name="CopyBuildOutputToTempOutput"
           AfterTargets="CopyFilesToOutputDirectory">
+
+    <ItemGroup>
```

Comment:
This removes the .cs file from the Compile sources. If it's included here, the output gets copied to the parent directory of the .cs file as part of this target.

Reply:
Yes, that's the "binplacing" target. It currently depends on the a single compile item.

To make this more robust, it would be better to change all usages of the `Compile` item to another item that only contains the actual compile item.

So: L129 & 133 shouldn't use `@(Compile)` anymore but another item.

Reply:
so something like this:

```
<ItemGroup>
  <CompileForRelativePath Include="@(Compile)" />
  <CompileForRelativePath Remove="$(RepoRoot)src\AssemblyInfo.cs" />
</ItemGroup>
```

Reply:
Thanks for the suggestion! Added here: https://github.com/dotnet/source-build-reference-packages/pull/794/commits/60266f871372130df00b5cf104002e5951aca8b1#diff-281c53fe4841f457012b2ddead53ad727b69e27c1f6d4911f2ada3042d035af0R125